### PR TITLE
feat(lte): Switch EPS Auth service to use S6a proxy proto

### PR DESCRIPTION
**Pull request by emakeev**
Signed-off-by: Evgeniy Makeev <evgeniym@fb.com>

## Summary

Switch EPS Auth service to use S6a proxy proto

A dedicated EAP Auth gRPC definition requires a significant overhead for AGW mme integration. Switching to already used by AGW S6a Proxy API would enable smother integration.

## Test Plan

Unit tests.

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: lte/cloud/go/services/eps_authentication/servicers/ai.go [hunk 1]
```diff
@@ -24,6 +24,7 @@ import (
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 
+	fegrotos "magma/feg/cloud/go/protos"
```
Comment (ssanadhya) [inline lte/cloud/go/services/eps_authentication/servicers/ai.go]:
```suggestion
	fegprotos "magma/feg/cloud/go/protos"
```

File: lte/cloud/go/services/eps_authentication/servicers/eps_auth_suite_test.go [hunk 2]
```diff
@@ -20,6 +20,7 @@ import (
 	"github.com/stretchr/testify/suite"
 	"golang.org/x/net/context"
 
+	fegprotos "magma/feg/cloud/go/protos"
 	"magma/lte/cloud/go/lte"
 	lteprotos "magma/lte/cloud/go/protos"
```
Comment (ssanadhya) [inline lte/cloud/go/services/eps_authentication/servicers/eps_auth_suite_test.go:25]: lteprotos can be removed

**Review by ssanadhya — APPROVED** [review]
LGTM, with minor nits

File: lte/cloud/go/services/eps_authentication/servicers/ai.go (hunk 1 above)
Comment (emakeev) [reply lte/cloud/go/services/eps_authentication/servicers/ai.go]: 👍

File: lte/cloud/go/services/eps_authentication/servicers/eps_auth_suite_test.go (hunk 2 above)
Comment (emakeev) [reply lte/cloud/go/services/eps_authentication/servicers/eps_auth_suite_test.go:25]: I think, addSubscriber uses lteprotos

**Review by uri200 — APPROVED** [review]

**Comment by tmdzk** [comment]
Force merging due to broken OAI tests

**Pull request merged**